### PR TITLE
chore(api): publish organizations_sessions endpoint

### PR DESCRIPTION
**Pull request by obostjancic**
Adds `organizations_sessions` to open api docs

image

### CODE REVIEW

File: api-docs/openapi.json [hunk 1]
```diff
@@ -108,9 +108,6 @@
     "/api/0/organizations/{organization_slug}/repos/{repo_id}/commits/": {
       "$ref": "paths/organizations/repo-commits.json"
     },
-    "/api/0/organizations/{organization_slug}/sessions/": {
```
Comment (sentaur-athena) [inline api-docs/openapi.json:111]: 🎉

File: src/sentry/api/serializers/sessions.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from typing import Dict, List
+
+from typing_extensions import TypedDict
+
+DateString = str
+
+
+class QuerySessionGroup(TypedDict):
```
Comment (sentaur-athena) [inline src/sentry/api/serializers/sessions.py:8]: Let's make sure these are also used in the implementation of the API. So in future if something changes in the response it reflect documentation too. For example is this the return type from `release_health.backend.run_sessions_query` same as `QuerySessionsResponse`? If so let's add a return type.

**Comment by sentaur-athena** [comment]
Please follow these steps to check the UI locally: https://develop.sentry.dev/api/public/#building-and-testing-locally
And add a screenshot to the PR. It'll make the review much easier.

File: src/sentry/api/serializers/sessions.py (hunk 2 above)
Comment (obostjancic) [reply src/sentry/api/serializers/sessions.py]: Great idea! I reused `SessionsQueryResult` type and removed `QuerySessionsResponse` in [b9a8fc1](https://github.com/getsentry/sentry/pull/64032/commits/b9a8fc15bfb1fdec0051ad091c741357522efbf4)

**Comment by obostjancic** [comment]
> Please follow these steps to check the UI locally: [develop.sentry.dev/api/public/#building-and-testing-locally](https://develop.sentry.dev/api/public/#building-and-testing-locally) And add a screenshot to the PR. It'll make the review much easier.

I Updated the PR description 🚀

**Review by matejminar — APPROVED** [review]

**Pull request merged**